package.json

```diff
@@ -30,28 +30,28 @@
   },
   "scripts": {
     "start": "start-storybook -p 9001 -c storybook -s ./storybook/static",
-    "build": "lerna run --parallel build -- -- --env production && npm run ts:emit",
+    "build": "lerna run --parallel build -- -- --env production && yarn ts:emit",
     "build:storybook": "build-storybook -c storybook -o ./storybook/public -s ./storybook/static",
-    "format": "npm run prettier && npm run eslint:format",
-    "init": "npm install && lerna bootstrap && npm run build",
-    "link-all-packages": "lerna exec --ignore @leafygreen-ui/lib --ignore @leafygreen-ui/theme -- npm link ",
+    "format": "yarn prettier && yarn eslint:format",
+    "init": "yarn && lerna bootstrap && yarn build",
```

Comment:
I think we can omit bootstrap now. If you test it, it should be fine.

Reply:
👍 you're right